MediaBrowser.Controller/MediaEncoding/EncodingJobInfo.cs

```diff
@@ -404,7 +404,10 @@ public bool EnableBreakOnNonKeyFrames(string videoCodec)
                 {
                     // Don't exceed what the encoder supports
                     // Seeing issues of attempting to encode to 88200
-                    return Math.Min(44100, BaseRequest.AudioSampleRate.Value);
+                    // return Math.Min(44100, BaseRequest.AudioSampleRate.Value);
+
+                    // I don't see any reason why limiting the sample rate to a maximum of 44100 !
+                    return BaseRequest.AudioSampleRate.Value;
```

Comment:
Seems like this should be the default value, otherwise wouldn't it have issues when no sample rate is set?

Reply:
This piece of code is under condition : if (BaseRequest.AudioSampleRate.HasValue).

- AudioSampleRate.HasValue == true and the corresponding value has been explicitly set before (with a CodecProfile for example), and it is exactly the value we want.
- AudioSampleRate.HasValue == false and the corresponding value will be defaulted later (with the default codec value).

the line Math.Min(44100, BaseRequest.AudioSampleRate.Value); does not address default value, but the maximum value. In other words, any value greater than 44100 becomes exactly 44100. It is very annoying when dealing with High Resolution Audio format (example : HD FLAC 96 khz), they are always down sampled to 44.1khz because of this.

Perhaps, you have a test case with which this change is not acceptable ? I don't see anyone.

Reply:
The comment above says `Seeing issues of attempting to encode to 88200`, so maybe when it was originally written there was an encoder limit